components/rating/index.css

```diff
@@ -247,7 +259,6 @@
 
 @media (forced-colors: active) {
```

Comment:
I did open up Assistiv Labs to check WHCM this time around. There is an S1 WHCM spec linked in the guidelines, and I'm wondering after looking at that and the guidelines if we've got this correct. The colors are off, and I would guess we would want the outline to be 2px (or focus-indicator width) instead of 1px.

That said, I wouldn't consider that a blocker here! I noted on the ticket for WHCM specs that we'd probably want some updated specs for Rating, and we could address that later. For now, I don't see anything low-contrast that seems like it would be an issue.